packages/astro/package.json

```diff
@@ -83,7 +83,7 @@
     "@clerk/shared": "2.5.2",
     "@clerk/types": "4.14.0",
     "nanoid": "5.0.7",
-    "nanostores": "0.10.3",
+    "nanostores": "0.11.2",
```

Comment:
Version bump with bug fixes and perf improvements. No breaking change. See [changelog](https://github.com/nanostores/nanostores/releases/tag/0.11.0)